packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -283,6 +283,26 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
     this._onKeyDown(this._onSelectNextMonth, ev);
   }
 
+  private _formatMonthYear(navigatedDate: Date) {
+    let { strings, formatDate } = this.props;
+    let formatedDate = strings.months[navigatedDate.getMonth()] + ' ' + navigatedDate.getFullYear();
```

Comment:
This can go on an else block so it doesn't get computed when a formatting function is present.

Reply:
And it looks like you don't need the strings variable in there (and in the other function as well).